cardinal/server/shutdown.go

```diff
@@ -0,0 +1,50 @@
+package server
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"pkg.world.dev/world-engine/cardinal/ecs"
+)
+
+type ShutDownManager struct {
```

Comment:
nit: You have a mix of "ShutDown" and "Shutdown" in this code. It should be consistent. "Shutdown" looks more natural to me, but either is fine.

Reply:
This has been changed to GameManager. @smsunarto mentioned using another less specific name here. I think game manager is the most clear while keeping the functionality general. In the future we can have this object manage all the things that need to coordinate world and server.